corehq/apps/ota/views.py

```diff
@@ -57,7 +57,7 @@
 
 @location_safe
 @handle_401_response
-@mobile_auth_or_token
+@mobile_auth_or_formplayer
 @check_domain_migration
 def restore(request, domain, app_id=None):
```

Comment:
@wpride what params get pass here in the request doing the restore for SMS?

Reply:
Sorry I don't fully understand the question - URL params? The set should be the same as for normal Web Apps restores

Reply:
Sorry, I should have been more specific. Will the `as_user` param always be passed?

Reply:
For SMS the `as-user` param will always be present in this view